.github/workflows/actions_build.yml

```
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-12, windows-latest]
-        java: [17]
+        java: [17, 21]
```

I prefer only 21 is run on mac/windows. The build is flaky enough as it is already 😅 

```suggestion
        java: [21]
```